package.json

```diff
@@ -16,7 +16,7 @@
     "access": "public"
   },
   "engines": {
-    "node": "14 || 16 || 17 || 18 || 19"
+    "node": "14 || 16 || 17 || 18 || 19 || 20"
```

Comment:
@achrinza any reason why you can't put `>=14` here so it will work for **21**, **22**, ... too?

Reply:
Hi Jokay,

It was decided to explicitly list tested Node.js versions to give users a general recommendation on which Node.js versions are officially supported by the package.

Although some details are outdated (e.g. we now test against non-LTS versions), the rationale behind this decision can be found at https://github.com/achrinza/node-ipc/issues/29#issuecomment-1084136491.

Hope it helps!

Reply:
Well, thank you for the clarification.